src/Components/ADT3DSceneBuilder/Internal/VisualRuleForm/VisualRuleForm.state.ts

```diff
@@ -33,11 +33,12 @@ export const VisualRuleFormReducer: IVisualRuleFormReducerType = produce(
         );
         switch (action.type) {
             case VisualRuleFormActionType.FORM_VISUAL_RULE_DISPLAY_NAME_SET: {
-                draft.visualRuleToEdit.displayName = action.payload.name.trim();
```

Comment:
how come we remove the trim here?

Reply:
Trim here doesn't allow blank space inputs. If I want my label to be `New label` the space would be trimmed every time it is the last character.